sdk/core/azure-core/src/cryptography/sha_hash.cpp

```diff
@@ -52,7 +53,7 @@ class SHAWithOpenSSL final : public Azure::Core::Cryptography::Hash {
   {
     if (1 != EVP_DigestUpdate(m_context, data, length))
     {
-      throw std::runtime_error("Crypto error while updating Sha256Hash.");
+      throw std::runtime_error("Crypto error while updating Sha256Hash."); // LCOV_EXCL_LINE
```

Comment:
We could use mocks or trampolines to test these lines rather than excluding them from coverage.

Reply:
Interesting idea. Not 100% sure how I tunnel a mocked EVP into SHA1 though. I'll have to investigate.